libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/AuditEvent.java

```diff
@@ -33,13 +33,19 @@ public class AuditEvent {
     @JsonInclude(JsonInclude.Include.NON_NULL)
     private final AuditRestricted restricted;
 
+    @JsonProperty("reprove_identity")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private final Boolean reproveIdentity;
```

Comment:
Are you sure this should go on the top-level audit event, as opposed to the `extensions`? Usually our custom fields go into the extensions object, which can vary across events.